o-fish-ios/Views/PatrolBoat/PatrolBoatView.swift

```diff
@@ -34,6 +37,11 @@ class DutyState: ObservableObject {
         }
         dutyChangeViewModel.user = user
         dutyChangeViewModel.status = onDuty ? .onDuty : .offDuty
+        if onDuty {
```

Comment:
is this a correct place when the duty synchronise?

Reply:
It's called before duty state will be saved (next line). So, I guess, It's correct place.